script/_common

```diff
@@ -1,3 +1,4 @@
+# shellcheck shell=bash
```

Comment:
Alternatively, this could be a shebang, wasn't sure which was more idiomatic given this is a lib file not intended to be directly executed...

Reply:
Not sure myself either but hinting spellcheck seems fine to me 👍.